packages/in_app_purchase/integration_test/in_app_purchase_test.dart

```diff
@@ -1,6 +1,7 @@
 // Copyright 2019, the Chromium project authors.  Please see the AUTHORS file
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
+// @dart = 2.10
```

Comment:
Any reason you're using 2.10, rather than the 2.9 that's suggested in the unsound NNBD docs?

Reply:
We used 2.9 in the rest of the plugins, so I'm not sure if I suggested 2.10

Reply:
Haha thanks! I was probably dreaming then. Will fix.